apps/backend/src/config/config.service.ts

```diff
@@ -57,10 +57,20 @@ export class ConfigService {
       oidcName: this.get('OIDC_NAME') || '',
       ldap: this.get('LDAP_ENABLED')?.toLocaleLowerCase() === 'true' || false,
       registrationEnabled: this.isRegistrationAllowed(),
-      localLoginEnabled: this.isLocalLoginAllowed()
```

Comment:
make sure to add these new envvars everywhere - here, in the docs, in the .env-example, in the helm chart